proarc-common/src/main/resources/cz/cas/lib/proarc/common/config/proarc.properties

```diff
@@ -154,6 +154,12 @@ export.kramerius4.rename.THUMBNAIL=IMG_THUMB
 # default access policy used for exported object. Values: policy:private|policy:public|<empty>
 export.kramerius4.policy=policy:private
 
+#subject that create the file (record]
```

Comment:
Snaž se udržet konzistentní formát `# subject`.
Co znamená `(record]`?
Překlep `that create/s/`.